File: Add Code Here/TYPESCRIPT/MedianOfTwoSortedArrays.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    const a = [...nums1, ...nums2]
    .sort((a_,b_) => a_-b_)
   
   if(a.length % 2){
     return a[((a.length-1)/2)]
   }
   return (a[(a.length/2) - 1] + a[((a.length/2))])/2;
};

File: Add Code Here/C++/Median of Two Sorted Arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        auto vec1=nums1.begin();
        auto vec2=nums2.begin();
        vector<int> ans;
        while(vec1!=nums1.end() || vec2!=nums2.end()){
            if(vec1==nums1.end()){
                ans.push_back(*vec2);
                vec2++;        
            }
            else if(vec2==nums2.end()){
                ans.push_back(*vec1);
                vec1++;
            }
            else{
                if(*vec1>*vec2){
                    ans.push_back(*vec2++);
                }
                else{
                    ans.push_back(*vec1++);
                }
            }     
        }
        
        if(ans.size()%2!=0) return ans[ans.size()/2];
        else    return (ans[ans.size()/2-1]+ans[(ans.size()/2)])/2.0;
    }
};

static const int _ = []() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    cout.tie(nullptr);
    return 0;
}();
